### ISSUE TITLE
Tooltip on a button problem

### ISSUE THREAD

**Issue by LucaCris** [vikramlearning/blazorbootstrap#226]
Surround a button with a tooltip:

```
<ToolTip Title=...>
            <Button Color="ButtonColor.Primary" @onclick="OnRefresh"><Icon Name="IconName.ArrowClockwise" /></Button>
</ToolTip>
```

The tooltip appears, but if the button action fire up a content redraw, the tooltip remain stucked on page (until click somewhere else).

Severity: **can't use tooltip in such buttons.**

**Comment by LucaCris** [comment]
![Screenshot 2023-06-08 112652](https://github.com/vikramlearning/blazorbootstrap/assets/50824617/b932db4e-d153-45d7-abf2-edd3bb76f932)

I confirm it doesn't work either...

`<Button Color="ButtonColor.Primary" @onclick="OnRefresh" TooltipTitle="Tooltip text" TooltipPlacement="TooltipPlacement.Top"><Icon Name="IconName.ArrowClockwise" /></Button>`

**Comment by gvreddy04** [comment]
@LucaCris This is fixed part of **Button TooltipTitle does not disappear when clicked #222**

**Comment by gvreddy04** [comment]
This is fixed part of #222.
